styles/globals.css

```diff
@@ -13,32 +13,35 @@
 
   --background-color: var(--grey-0);
   --background-card: #141415;
+  --background-card-hover: #18181b;
   --main-color: var(--yellow-0);
   --main-color-dark: var(--yellow-1);
   --font-color-strong: var(--white-0);
   --font-color-default: var(--white-1);
   --border-color: var(--white-2);
 }
 
+html {
+  color-scheme: dark;
+}
+
+html,
+body,
+#__next {
+  height: 100%;
+}
+
 body {
   background-color: var(--background-color);
   color: var(--font-color-default);
+  overflow-y: scroll; /* avoid layout shift on pages that don't scroll */
 }
 
-main,
-footer,
-#__next > header {
-  padding: 8px 20px;
-  max-width: 1200px;
-  margin: auto;
-}
-
-.highlight {
-  display: inline;
-  padding: 0.25em 0;
-  background: var(--main-color);
-  color: var(--background-color);
-  box-shadow: 0.5em 0 0 var(--main-color), -0.5em 0 0 var(--main-color);
+#__next {
+  display: grid;
+  grid-template: auto 1fr auto / 100%;
+  width: min(90%, 1100px);
```

Comment:
fluid width instead of fixed padding, the diff is mostly noticeable on medium/tablet screens (~ 800px - 1200px)